tests/__tests__/api/issue_map/patchIssueMap.js

```diff
@@ -0,0 +1,212 @@
+    const errorMessage = JSON.parse(response.text).error;
+    expect(errorMessage).toStrictEqual("Invalid request body.");
+  });
+
+  test("Invalid article addition due to article slug already existed", async () => {
+    const requestBody = {
+      articleSlug: validArticleSlug,
+      issueNumber: 1,
+      pageLength: 3,
+      authors: ["ethan@ethan.com"],
+      editors: ["ethan@ethan.com"],
+      designers: ["ethan@ethan.com"],
+      photographers: ["ethan@ethan.com"],
+      section: validSectionName,
+      categories: "biology",
+    };
+    const response = await request(app)
+      .patch(`/issue-map/add-and-create-article`)
+      .set("Cookie", [`token=${tokens["ethan@ethan.com"]}`])
+      .send(requestBody);
+    showLog && console.log(response);
+    expect(response.status).toBe(404);
+    const errorMessage = JSON.parse(response.text).error;
+    expect(errorMessage).toStrictEqual("Invalid request body.");
+  });
+
+  test("should create and add article to issue map successfully with missing authors/editors/designers/photographers", async () => {
+    const requestBody = {
+      articleSlug: "New article",
+      issueNumber: 2,
+      pageLength: 5,
+    };
+
+    const response = await request(app)
+      .patch(`/issue-map/add-and-create-article`)
+      .set("Cookie", [`token=${tokens["ethan@ethan.com"]}`])
+      .send(requestBody);
+
+    console.log(response.body);
+    expect(response.status).toBe(200);
+    const createdArticleID = response.body.articles.at(-1);
+    const article = await ArticlesAccessor.getArticle(createdArticleID);
+    console.log(article);
+    expect(article.issueNumber).toStrictEqual(requestBody.issueNumber);
+    expect(article.slug).toStrictEqual(requestBody.articleSlug);
+  });
+
+  test("should create and add article to issue map successfully with authos/editors/designers/photographers given", async () => {
+    const requestBody = {
+      articleSlug: "Another new article",
+      issueNumber: 2,
+      pageLength: 3,
+      authors: ["ethan@ethan.com"],
+      editors: ["ethan@ethan.com"],
+      designers: ["ethan@ethan.com"],
+      photographers: ["ethan@ethan.com"],
+      section: validSectionName,
+      categories: "biology",
+    };
+
+    const response = await request(app)
+      .patch(`/issue-map/add-and-create-article`)
+      .set("Cookie", [`token=${tokens["ethan@ethan.com"]}`])
+      .send(requestBody);
+
+    console.log(response.body);
+    expect(response.status).toBe(200);
+
+    const sectionIndex = response.body.sections.findIndex((sec) => sec.sectionName === validSectionName);
+
+    const createdArticleID = response.body.sections[sectionIndex].articles.at(-1);
+    console.log(response.body.sections[sectionIndex]);
+
+    const article = await ArticlesAccessor.getArticle(createdArticleID);
+    console.log(article);
+
+    expect(article.issueNumber).toStrictEqual(requestBody.issueNumber);
+    expect(article.slug).toStrictEqual(requestBody.articleSlug);
```

Comment:
Same here - consider adding a few more .toStrictEqual checks here to confirm that designStatus is set to Has_Designer, photographyStatus is set to Has_Photographer, etc